### ISSUE TITLE
OverviewMap issues

### ISSUE THREAD

**Issue by bstroebl** [qgis/qwc2-demo-app#95]
Where is the OverviewMap configured?
Where can the tooltip _OverviewMap_ be translated?

**Comment by manisandro** [comment]
I've added some examples of passing options to map support tools in the demo config.json [1]. Supported options are those of the stock OpenLayer controls, so for the OverviewMap supported options would be those documented at [2].

[1] https://github.com/qgis/qwc2-demo-app/blob/master/config.json#L180
[2] https://openlayers.org/en/latest/apidoc/ol.control.OverviewMap.html

**Comment by bstroebl** [comment]
Hi Mani,
basically I would like to tell the application to use a certain WMS layer as overview map. What happens currently is that the current base layer is requested (seen in my app and according to [2]). I could live with this however the overview map is empty. I dug into the requests sent and discovered the following:
The WMS-Request says VERSION=1.3.0 however it contains both, a CRS and a SRS parameter. Both differ: CRS=EPSG:3857, SRS=EPSG:25832 (my project's projection); the BBOX sent seems to be in EPSG:25832. If I manually change CRS to EPSG:25832 a valid map is returned.
Basically what is needed AFAIU is that the OL default EPSG:3857 is replaced by the current CRS.
What would be desirable is to manually set the overview map to a certain data source (optional).
I hope this is understandable.

**Comment by manisandro** [comment]
Does https://github.com/qgis/qwc2/commit/d63a7c0e94fbeab4b5465979f5adb2e18e6591c0 help with the CRS/SRS issue?

Selecting which layers to show in the overview map is currently not possible and would need to be implemented.

**Comment by bstroebl** [comment]
> Does qgis/qwc2@d63a7c0 help with the CRS/SRS issue?

No, unfortunately not. Still SRS is EPSG:25832 and CRS is EPSG:3857

**Comment by manisandro** [comment]
At long long last fixed with

https://github.com/qgis/qwc2/commit/abe40f2a9166a27886607c8f205e38aa6e5c3d79
